lib/app_config.rb

```diff
@@ -16,32 +16,30 @@ def self.require_keys(keys)
   class Environment
     attr_reader :config
 
-    def initialize(configuration, env)
+    def initialize(configuration)
       @config = {}
 
-      keys = Set.new(configuration.keys - %w[production development test])
-      keys |= configuration[env]&.keys || []
+      keys = configuration.keys
 
       keys.each do |key|
-        value = configuration.dig(env, key) || configuration[key]
-        env_value = ENV[key]
+        value = configuration[key]
+        env_value = ENV[key.to_s]
 
-        check_string_key(key)
         check_string_value(key, value)
 
         if env_value
           warn "WARNING: #{key} is being loaded from ENV instead of application.yml"
           @config[key] = env_value
         else
           @config[key] = value
-          ENV[key] = value
+          ENV[key.to_s] = value
```

Comment:
I *think* soon we can remove this env var setting. maybe not in this PR, but since we're here, it reminded me

Reply:
Yeah, I'm not sure any tooling _actually_ depends on it. I think when we can leave this behind once we drop this on the move the IdentityConfig.